dev/app.js

```diff
@@ -226,6 +227,14 @@ renderSiteApp(
                     pathname: "/line/disable-calibration",
                     content: renderDisableCalibration,
                     title: "Disable Calibration"
+                },
+                {
+                    pathname: "/line/graph-and-legend-padding-reduced",
+                    content: (id) => {
+                        createElementLegendBindTo(id);
+                        return renderGraphAndLegendPaddingReduced(id);
+                    },
+                    title: "Graph and Legend Padding Reduced"
                 }
```

Comment:
Can we add this to all graph types as well?

Reply:
Added [here](https://github.com/JananiGunasekaran/carbon-graphs/commit/1ab1712b764982cc553e4b766ec58506b272af2f).